<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RufusWindow</class>
 <widget class="QMainWindow" name="RufusWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rufusl 0.1</string>
  </property>
  <widget class="QWidget" name="container">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="deviceLabel">
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="slavePartition">
        <item>
         <widget class="QLabel" name="partitionLabel">
          <property name="text">
           <string>Partition scheme and target system type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="partitionCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="slaveFs">
        <item>
         <widget class="QLabel" name="fsLabel">
          <property name="text">
           <string>File system</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fsCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="slaveCluster">
        <item>
         <widget class="QLabel" name="clusterLabel">
          <property name="text">
           <string>Cluster size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="clusterCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="slaveVolume">
        <item>
         <widget class="QLabel" name="volumeLabel">
          <property name="text">
           <string>New volume label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="volumeEdit">
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="settingsMaster">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #d2d2d2;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
        </property>
        <property name="title">
         <string>Format options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="badBlocks">
           <item>
            <widget class="QCheckBox" name="badBlocksCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="statusTip">
              <string>Not implemented.</string>
             </property>
             <property name="text">
              <string>Check device for bad blocks</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="badBlocksPass">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>162</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="formatCheck">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string>Quick format</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="usingCont">
           <item>
            <widget class="QCheckBox" name="usingCheck">
             <property name="text">
              <string>Create bootable image using</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sourceCombo"/>
           </item>
           <item>
            <widget class="QPushButton" name="usingSearch">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="labelCheck">
           <property name="text">
            <string>Create extended label and icon files</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="slaveProgress">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="statusEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>READY</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="slaveButtons">
        <item>
         <widget class="QPushButton" name="buttonAbout">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="buttonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStart">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClose">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
